modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/accessor/BucketAccessor.java

```diff
@@ -45,6 +46,9 @@
     @Query("select * from bucket where experiment_id = ?")
```

Comment:
Need to keep this method  as it is still being used in other places other than batch-assignment.